webapp/loggedin/surveyViews/taxonomyEdit/components/taxaTableRow.js

```diff
@@ -16,9 +17,12 @@ const TaxaTableRow = props => {
       <div>{Taxon.getScientificName(taxon)}</div>
       {
         vernacularLanguageCodes.map(lang => {
-            const vernacularName = Taxon.getVernacularNameByLang(lang)(taxon)
-            return <div key={TaxonVernacularName.getUuid(vernacularName)}>
-              {TaxonVernacularName.getName(vernacularName)}
+            const vernacularNames = Taxon.getVernacularNamesByLang(lang)(taxon)
+            return <div key={`${Taxon.getUuid(taxon)}_vernacular_name_${lang}`}>
+              {R.pipe(
+                R.map(TaxonVernacularName.getName),
+                R.join(' / ')
```

Comment:
Should this use the constant defined in another file as the separator?

Reply:
Uhm, rigth, so it will be consistent with the one used in data import/export